organisms/organisation-unit-tree/new-features/select-by-group-level.feature

```diff
@@ -0,0 +1,56 @@
+# Generic
+
+Feature: Selection of descendant levels and/or groups inside an org unit
```

Comment:
Although conceptually very similar, I still think that `Selection of descendant levels` and `Selection of descendant groups` should be two different feature files as it's basically two different features.
There will be a lot of repetition (which is good! Because we can reuse the implementation for identical sentences in the tests)

Reply:
Ok - I'll go with that. It feels a little funky for me to do this, anti-DRY! But I see how it makes sense to split these files.